include/dem/distributions.h

```diff
@@ -31,20 +31,24 @@ class Distribution
   /**
    * @brief Carries out the size sampling of particles. This is the base class of
    * NormalDistribution, UniformDistribution, LogNormalDistribution and
```

Comment:
Is the "LogNormalDistribution" class actually implemented?

```suggestion
   * NormalDistribution, UniformDistribution and CustomDistribution classes.
```

Reply:
Good catch. It was, but I have decided to implement custom first then log-normal, otherwise this pull request would have been to big